File: test/httpclient.py
#coding=utf-8

import http.client,urllib.parse


def post_demo():
    params = urllib.parse.urlencode({'tokenId': 0, 'minter': '3423412dfasf','owner':'123adfs','collection':'asdfasdfasd','txhash':'asdfasdfasd'})
    headers = {"Content-type": "application/json",
               "Accept": "application/json"}
    conn = http.client.HTTPSConnection("backend.ethmore.pro")
    conn.request("POST", "/api/v1/setNftData", params, headers)
    response = conn.getresponse()
    print(response.status, response.reason)

    if not response.closed:
        data = response.read()
        print(data, type(data.decode('utf-8')))

    conn.close()


if __name__=='__main__':
    post_demo()

File: test/httpclient.js
const axios=require('axios');

async function post(){
    let res = await  axios({
        url: 'https://backend.ethmore.pro/api/v1/setNftData', // 向url发送请求
        method: 'post', // 指定psot请求
        data:{'tokenId': 0, 'minter': '3423412dfasf','owner':'123adfs','collection':'asdfasdfasd','txhash':'asdfasdfasd'}, // 携带数据
    })
    console.log(res.data) // 后端传回的数据
}

post();
